README.md

```diff
@@ -289,8 +289,12 @@ $ cd carbon-lang
 $ bazel run //explorer -- ./explorer/testdata/print/format_only.carbon
 ```
 
-These instructions assume [Homebrew](https://brew.sh/) is installed; see our
-[contribution tools documentation](/docs/project/contribution_tools.md) for more
+These instructions assume [Homebrew](https://brew.sh/) is installed;
+Notice that: 
+- Homebrew is available only on Linux and macOS
+- If you are a Windows user, you can install Linux as a subsystem (https://youtu.be/AaZUJUbXrQ8)
+
+see our [contribution tools documentation](/docs/project/contribution_tools.md) for more
```

Comment:
Actually, we aren't encouraging Homebrew on Linux anymore. We should probably rewrite these instructions to assume that a recent version of llvm is available, and leave other assumptions for contribution_tools.md.

If you'd like to add Windows advice, I'm going to request it in textual form on contribution_tools.md. We've been trying to write out instructions, and I don't think a video will work for everyone.

Reply:
#2637 to see about removing the notes on brew.